Formula/qjackctl.rb

```diff
@@ -14,14 +14,23 @@ class Qjackctl < Formula
   depends_on "autoconf" => :build
   depends_on "qt5"
   depends_on "jack"
+  depends_on "portaudio" => :linked
+
+  # Fixes varaible length array error with LLVM/Clang
+  # Reported upstream: https://github.com/rncbc/qjackctl/issues/17
+  patch do
+    url "https://github.com/rncbc/qjackctl/pull/18.patch"
```

Comment:
I had to look up how to do that, but I guess this is now accomplished in the latest commit of the formula (d9a396e).